source/standards/vulnerability-disclosure.html.md.erb

```diff
@@ -31,7 +31,12 @@ You should use <https://vdp.cabinetoffice.gov.uk/.well-known/security.txt>
 in either:
 
 - the origin for your site's `/.well-known/security.txt`
-- the destination of a 302 redirect for `/.well-known/security.txt`
+- the destination of a redirect for `/.well-known/security.txt`
+  - note - try implementing the redirect mechanism in the following order:
+    1. Server-side redirect (302 status and `Location` header in response)
```

Comment:
have we checked what happens when this renders? I know some Markdown processors may have problems with newlines being not present between things, but could be it's safe as it's inside a list

Reply:
I had not 🤦‍♂️ Made changes, here's how it looks:

![image](https://user-images.githubusercontent.com/5426038/140395869-897d7191-798d-470d-b605-3e105e24f878.png)